src/main/java/com/amplitude/Amplitude.java

```diff
@@ -98,6 +113,10 @@ private void tryToFlushEventsIfNotFlushing() {
     }
   }
 
+  private Response getHttpCall(List<Event> eventsInTransit) {
```

Comment:
I see, originally I was suggesting to use `getHttpCall` to return the instance of `HttpCall`, if you want to make a call in this method, then maybe give a different name